bot.py

```diff
@@ -43,6 +43,7 @@
     InvalidConfigError,
     PermissionLevel,
     SafeFormatter,
+    configure_log_format,
```

Comment:
This is not used anywhere in this file, right?

Reply:
Thought I removed that, removed.